recipe/meta.yaml

```
@@ -68,11 +67,11 @@ outputs:
         - wheel
       run:
         - python
-        - packaging
-        - pandas ==1.5.3
+        - packaging >=21.0
+        - pandas >=2.1,<2.2
         - numpy >=1.18.5
```

```suggestion
        - numpy >=1.22.4
```
https://github.com/modin-project/modin/blob/0.26.1/setup.py#L57